src/libs/CardUtils.ts

```diff
@@ -451,6 +451,28 @@ const getDescriptionForPolicyDomainCard = (domainName: string): string => {
     return domainName;
 };
 
+const CUSTOM_FEEDS = [CONST.COMPANY_CARD.FEED_BANK_NAME.MASTER_CARD, CONST.COMPANY_CARD.FEED_BANK_NAME.VISA, CONST.COMPANY_CARD.FEED_BANK_NAME.AMEX];
+
+function getFeedType(cardFeed: CompanyCardFeed, cardFeeds: OnyxEntry<CardFeeds>): CompanyCardFeedWithNumber {
```

Comment:
I find it confusing that the cardFeed is used here for the same that feedType represents.

Reply:
no, CompanyCardFeed type is a value of FEED_BANK_NAME, so it's only a name. I can rename this to feedName if you prefer

Reply:
Yes please 🙌 or do you have any concerns? maybe I am reading it wrong

Reply:
maybe one 😅 - we use feedName as the full name of the feed like here. TBH I can't come up with a better name - if we send feedType but pass a param of CompanyCardFeed type what better name can we think of here?
<img width="683" alt="Screenshot 2025-01-15 at 16 09 12" src="https://github.com/user-attachments/assets/d9fccafd-ba27-4779-b8eb-2c5aa0f67ddc" />

Reply:
FeedKey maybe?  as they are all mainly used as key in the settings object

Reply:
oki, done